src/lib_json/json_writer.cpp

```diff
@@ -1159,6 +1167,7 @@ StreamWriter* StreamWriterBuilder::newStreamWriter() const {
   bool eyc = settings_["enableYAMLCompatibility"].asBool();
   bool dnp = settings_["dropNullPlaceholders"].asBool();
   bool usf = settings_["useSpecialFloats"].asBool();
+  bool emitUTF8 = settings_["emitUTF8"].asBool();
```

Comment:
prefer const. Bonus points if you make all the other settings_ bools const as well.

Reply:
Done.